### ISSUE TITLE
Configuring panels and widgets (plasmashell) crashes the UI

### ISSUE THREAD

**Issue by ldorigo** [heckelson/i3-and-kde-plasma#8]
Hi,

Asking here as you may have experienced something similar: when I edit panels (the "menu bar" on the top or bottom of the screen, with the system tray etc.), the interface is very buggy, and if I click "Add Widgets" it usually crashes entirely. Any idea what I can do to fix that?

Thanks for the tutorial, other than a few minor quirks I'm really happy with it.

**Comment by WahidX** [comment]
Hi,
@ldorigo  did you find any solution to this cause I'm also facing the same issue.

**Comment by ldorigo** [comment]
No, but see my question on reddit for a little more information: https://old.reddit.com/r/i3wm/comments/lmhcy1/i3_plasma_add_widgets_crashes_plasma/

**Comment by ldorigo** [comment]
(oops, closed by mistake)

**Comment by Bad3r** [comment]
I don't think that there is a fix for this bug. The easiest solution is login into a regular plans session -> adding: editing widgets -> switch back to plasma+i3
